com.unity.cinemachine/Runtime/Deprecated/CinemachinePathBase.cs

```diff
@@ -111,13 +111,17 @@ public virtual float StandardizePos(float pos)
             float end = MaxPos;
             if (searchRadius >= 0)
             {
-                int r = Mathf.FloorToInt(Mathf.Min(searchRadius, (end - start) / 2f));
-                start = startSegment - r;
-                end = startSegment + r + 1;
-                if (!Looped)
+                if (Looped)
                 {
-                    start = Mathf.Max(start, MinPos);
-                    end = Mathf.Min(end, MaxPos);
+                    var r = Mathf.FloorToInt(Mathf.Min(searchRadius, (end - start) / 2f));
+                    start = startSegment - r;
+                    end = startSegment + r + 1;
+                }
+                else
+                {
+                    var r = Mathf.FloorToInt(Mathf.Min(searchRadius, end - start));
```

Comment:
what is the purpose of this line?  It seems to do nothing useful, given the next 2 lines